### PR TITLE
initial commit of win_dns_client module

### PR DESCRIPTION
**Pull request by nitzmahone**
##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
win_dns_client

##### SUMMARY
Module for managing DNS lookup configuration on Windows hosts.

### CODE REVIEW

File: lib/ansible/modules/windows/win_dns_client.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+short_description: Configures DNS lookup on Windows hosts
+description:
+  - The C(win_dns_client) module configures the DNS client on Windows network adapters.
+options:
+  adapter_names:
+    description:
+      - Adapter name or list of adapter names for which to manage DNS settings ('*' is supported as a wildcard value).
+        The adapter name used is the connection caption in the Network Control Panel or via C(Get-NetAdapter), eg C(Local Area Connection).
+    required: true
+  ipvr_addresses:
```
Comment (jhawkesworth) [inline lib/ansible/modules/windows/win_dns_client.py]: a teeny typo: ipvr_addresses -> ipv4_addresses

Comment (nitzmahone) [reply lib/ansible/modules/windows/win_dns_client.py]: dangit, good catch

**Comment by dagwieers** [comment]
Missing from CHANGELOG.md ! :-)

**Comment by nitzmahone** [comment]
I'm doing a pass over that once they're all in. :) Putting them in the PRs
is a recipe for endless merge conflicts.

**Pull request merged**